php-classes/Slate/Courses/SectionsRequestHandler.php

```diff
@@ -110,11 +111,52 @@ public static function handleRecordRequest(ActiveRecord $Section, $action = fals
                 ]));
             case 'students':
                 return static::handleStudentsRequest($Section);
+            case false:
```

Comment:
```suggestion
            case '':
            case false:
```

As can be found in the underlying `handleRecordRequest` implementation, we need both cases to catch `/blog/myblog` and `/blog/myblog/`